app/templates/administrative-units/administrative-unit/sites/site/edit.hbs

```diff
@@ -0,0 +1,169 @@
+<div class="au-c-body-container au-c-body-container--scroll">
+  <div class="au-o-box au-o-flow au-o-flow--large">
+    <div class="au-o-flow au-o-flow--tiny">
+      <AuHeading class="au-u-margin-bottom-tiny">
+        Bewerk vestiging
+      </AuHeading>
+      <AuHeading @level="2" @skin="3" class="au-u-margin-bottom-tiny">
+        {{@model.adminUnitSite.site.address.fullAddress}},
+        {{@model.adminUnitSite.administrativeUnit.name}}
+      </AuHeading>
+    </div>
+
+    <form
+      id="edit-vestiging-form"
+      aria-label="edit-vestiging-form"
```

Comment:
Is that something that's forced by a linting rule or something? First time I see it :smile:

Reply:
I use `Ctrl+Shift+I` to format

Reply:
I meant the aria-label :smile:

Reply:
It's like this in the core edit 😬

Reply:
ahh, we probably had multiple form elements on that page at one point. Anyhow, doesn't really matter.